src/script/view_model/ListViewModel.ts

```diff
@@ -144,8 +142,6 @@ export class ListViewModel {
     });
 
     this._initSubscriptions();
-
-    ko.applyBindings(this, document.getElementById(this.elementId));
```

Comment:
no more querying the DOM from the view model, the template takes care of binding things together (but this will be deleted soonish anyways ;) )